src/main/java/lotto/constant/Rank.java

```diff
@@ -0,0 +1,50 @@
+package lotto.constant;
+
+import java.util.Arrays;
+
+public enum Rank {
+    NO_PLACE("", 0, 0),
+    FIFTH_PLACE("3개 일치", 3, 5000),
+    FOURTH_PLACE("4개 일치", 4, 50000),
+    THIRD_PLACE("5개 일치", 5, 1500000),
+    SECOND_PLACE("5개 일치, 보너스 볼 일치", 6, 30000000),
+    FIRST_PLACE("6개 일치", 6, 2000000000);
```

Comment:
이렇게 숫자의 자리수가 많아질 경우 `_`으로 자리수 구분을 해 줄 수 있어요!
```suggestion
    FIRST_PLACE("6개 일치", 6, 2_000_000_000);
```

Reply:
저도 몰랐는데 하나 배워갑니다

Reply:
숫자에도 언더바를 사용할 수 있군요 훨씬 가독성이 좋아진 것 같아요!
덕분에 이번 프로젝트에서는 바로 적용했습니다ㅎㅎ 감사합니다!